src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -281,32 +285,64 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
+                } else {
+                    int page = args.length == 2 ? Integer.valueOf(args[4]) : 1;
+                    Set<String> groups = this.getPermissions(channel).getGroups().keySet();
+                    List<String> groupList = GeneralUtils.orderList(groups);
 
-                String list = getStringList(groupList, page);
-                EmbedBuilder eb = MessageUtils.getEmbed(sender);
-                eb.addField("Groups", list, false);
-                eb.addField("Current page", String.valueOf(page), true);
-                int pageSize = 20;
-                int pages =
-                        groups.size() < pageSize ? 1 : (groups.size() / pageSize) + (groups.size() % pageSize != 0 ? 1 : 0);
-                eb.addField("Pages", String.valueOf(pages), true);
-                eb.setColor(Color.CYAN);
-                channel.sendMessage(eb.build()).queue();
+                    String list = getStringList(groupList, page);
+                    EmbedBuilder eb = MessageUtils.getEmbed(sender);
+                    eb.addField("Groups", list, false);
+                    eb.addField("Current page", String.valueOf(page), true);
+                    int pageSize = 20;
+                    int pages =
+                            groups.size() < pageSize ? 1 : (groups.size() / pageSize) + (groups.size() % pageSize != 0 ? 1 : 0);
+                    eb.addField("Pages", String.valueOf(pages), true);
+                    eb.setColor(Color.CYAN);
+                    channel.sendMessage(eb.build()).queue();
+                    return;
+                }
+            } else if (args[0].equalsIgnoreCase("reset")) {
+                guild.setPermissions(new PerGuildPermissions());
+                MessageUtils.sendSuccessMessage("Successfully reset perms", channel, sender);
+                return;
+            } else if (args[0].equalsIgnoreCase("restoredefault")) {
+                if (guild.getPermissions().hasGroup("Default")) {
+                    guild.getPermissions().deleteGroup("Default");
+                }
+                guild.getPermissions().addGroup("Default");
+                Group defaults = guild.getPermissions().getGroup("Default");
+                for (Command command : FlareBot.getInstance().getCommands()) {
+                    if (command.isDefaultPermission()) {
+                        defaults.addPermission(command.getPermission());
+                    }
+                }
+                defaults.addPermission("flarebot.userinfo.other");
+                for (Member members : guild.getGuild().getMembers()) {
+                    guild.getPermissions().getUser(members).addGroup(defaults);
+                }
+                MessageUtils.sendSuccessMessage("Successfully restored the Default group", channel, sender);
                 return;
             }
         }
-        MessageUtils.sendUsage(this, channel, sender);
+        EmbedBuilder ufusage = getEmbedUsage(); //Un-Formatted Usage
```

Comment:
Inline this, only used once

Reply:
Yegh i was going to use that variable more but then i found out jda doesn't let you access all the things i wanted to access from EmbedBuilder.